src/js/gi/reducers/autocomplete.js

```diff
@@ -0,0 +1,41 @@
+import { AUTOCOMPLETE_TERM_CHANGED, AUTOCOMPLETE_STARTED, AUTOCOMPLETE_FAILED, AUTOCOMPLETE_SUCCEEDED } from '../actions';
+
+const INITIAL_STATE = {
+  inProgress: false,
+  previewVersion: null,
+  search_term: '',
+  facility_code: null,
```

Comment:
Why are these two keys snake_case while the first two are camelCase? In any case, linter will fail on the snake_cased keys, so they should be changed to camelCase.

If it's because the data received from the API would need to be transformed into the proper casing, you can actually just add the header `X-Key-Inflection: 'camel'` to the requests.